README.md

```diff
@@ -1,7 +1,82 @@
 # Open MCT
 
-Open MCT is a web-based platform for mission operations user interface
-software.
+Open MCT is a next-generation mission control framework being developed at the NASA Ames Research Center in Silicon Valley. Web-based, for desktop and mobile.
+
+Please visit our [Official Site](https://nasa.github.io/openmct/)
+
+##How is NASA Using Open MCT?
+
+Software based on Open MCT is being used for mission planning and operations in the lead up to the [Resource Prospector mission](https://www.nasa.gov/resource-prospector) and at NASA's Jet Propulsion Laboratory to view data from the Curiosity Rover.
+
+##Features
+
+ - Support planning and operations of any system that produces telemetry
+ -  Support space missions
+ - Visualize Data
+   - Streaming Data
+   - Historical Data
+   - Imagery
+   - Timelines
+   - Procedures
+   - etc.
+
+##See Open MCT in Action
+
+Do you want to see how Open MCT work? Try our demo.
+[LIVE DEMO](https://openmct-demo.herokuapp.com/)
+![Demo](https://nasa.github.io/openmct/static/res/images/Open-MCT.Browse.Layout.Mars-Weather-1.jpg)
+
+##Getting Started
+
+Here is how to run Open MCT in you local machine.
+
+1. Download or Clone Open MCT
+ `git clone https://github.com/nasa/openmct.git`
+
+2. Install Development Dependencies
+    `npm install`
+
+3. Run on your local machine
+    `npm start`
+
+4. Open your web browser and go to http://localhost:8080/
+
+5. Wait for splash screen finish then you should be able to play with Open MCT.
+
+For more information on building and running Open MCT, please see our getting started guide - https://nasa.github.io/openmct/getting-started/
+
+## Build
```

Comment:
This is confusingly titled (not your fault). This is really describing the steps necessary for deployment and is not necessary for development builds.

Could we rename to 

## Deploying Open MCT